### PR TITLE
Add WebGL2 tile to site

### PR DESCRIPTION
**Pull request by potch**

### CODE REVIEW

File: public_html/index.html [hunk 1]
```diff
@@ -35,6 +35,17 @@
         <h4 class="section-title">Top Stories</h4>
         <ul>
           <li class="highlight__item">
+            <a href="https://hacks.mozilla.org/2017/01/webgl-2-lands-in-firefox/" class="link">
+              <div class="imageWrapper">
+                <img src="assets/img/tile-webgl2.png">
+              </div>
+              <figcaption>
+                <h2 class="highlight__item__title">WebGL2 Lands in Firefox</h2>
+                <p>Tighten up the graphics on level 3 like never before</p>
```
Comment (cvan) [inline public_html/index.html]: heh, what's `level 3`?

File: public_html/index.html [hunk 2]
```diff
@@ -35,6 +35,17 @@
         <h4 class="section-title">Top Stories</h4>
         <ul>
           <li class="highlight__item">
+            <a href="https://hacks.mozilla.org/2017/01/webgl-2-lands-in-firefox/" class="link">
+              <div class="imageWrapper">
+                <img src="assets/img/tile-webgl2.png">
```
Comment (cvan) [inline public_html/index.html]: could use some `alt` text

Comment (cvan) [inline public_html/index.html]: by running `pngquant` on this PNG, I was able to get the size down from `559 KB` to `168 KB`. the compression's quite good; I can barely discern any difference.

**Comment by cvan** [comment]
r+wc

File: public_html/index.html (hunk 1 above)
Comment (potch) [inline public_html/index.html]: expecting to get a rewrite of this text, but [this](http://knowyourmeme.com/memes/tighten-up-the-graphics-on-level-3)

**Pull request merged**